sdk-autoconfigure-support/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+}
+
+group = "io.opentelemetry.javaagent"
+
+dependencies {
+  api("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
+
+  compileOnly("com.google.code.findbugs:annotations")
+  testCompileOnly("com.google.code.findbugs:annotations")
+
+  annotationProcessor("com.google.auto.service:auto-service")
+  compileOnly("com.google.auto.service:auto-service-annotations")
+  testCompileOnly("com.google.auto.service:auto-service-annotations")
```

Comment:
as far as I can tell `@AutoService` is not used in this module

Reply:
it is [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/3669dfb7a9203bd95cfa7c755c5e1bc1e92e8433/sdk-autoconfigure-support/src/main/java/io/opentelemetry/instrumentation/resources/ResourceProviderPropertiesCustomizer.java#L21) - but maybe an overkill just for one class